src/Components/Artwork/GridItem.tsx

```diff
@@ -44,37 +43,52 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
   private image: HTMLImageElement = null
 
   state = {
-    width: 1,
-    height: 1,
+    isMounted: false,
   }
 
   componentDidMount() {
-    const scale = window.devicePixelRatio
-    const width = this.image.width * scale
-    const height =
-      (this.image.width / this.props.artwork.image.aspect_ratio) * scale
-
     this.setState({
-      width,
-      height,
+      isMounted: true,
     })
   }
 
-  get imageURL() {
+  getImageUrl(breakpoints) {
     const imageURL = this.props.artwork.image.url
-    if (imageURL) {
+
+    if (!imageURL) {
+      return null
+    }
+
+    if (this.state.isMounted) {
+      const [[match]] = Object.entries(breakpoints).filter(([key, val]) => val)
+
+      const getWidth = () => {
+        switch (match) {
+          case "xs":
+          case "sm":
+          case "md":
+            return 400
+          case "lg":
+          case "xl":
+          default:
+            return 300
+        }
+      }
+
+      const scale = window.devicePixelRatio
+      const width = String(getWidth() * scale)
+      const height = String(
```

Comment:
should we maybe use `Math.floor` here so we don't have long decimals in the generated urls? 
<img width="325" alt="screen shot 2018-07-31 at 7 56 56 pm" src="https://user-images.githubusercontent.com/1457859/43493405-e779df6c-94fb-11e8-8ab4-148219e5d54f.png">